Source/Core/Core/DSP/DSPCore.h

```diff
@@ -308,6 +308,9 @@ struct SDSP
 
   // This one doesn't really belong here.
   u8* cpu_ram;
+
+  // This is used by DSPJitRegCache to store EBP.
+  u64 ebp_store;
```

Comment:
This should likely not be in the central core struct. If someone where to ever create a JIT for a different arch, this would be exposing unnecessary x86 JIT details.